satpy/modifiers/angles.py

```diff
@@ -183,6 +205,22 @@ def _cache_results(self, res, zarr_format):
         da.compute(new_res)
 
 
+def _get_output_chunks_from_func_arguments(args):
+    """Determine what the desired output chunks are.
+
+    It is assumed a tuple of tuples of integers is defining chunk sizes. If
+    a tuple like this is not found then arguments are checked for array-like
+    objects with a ``.chunks`` attribute.
+
+    """
+    chunked_args = [arg for arg in args if hasattr(arg, "chunks")]
+    tuple_args = [arg for arg in args if _is_chunk_tuple(arg)]
+    if not tuple_args and not chunked_args:
+        raise RuntimeError("Cannot determine desired output chunksize for cached function.")
```

Comment:
Is this line tested?

Reply:
No and it isn't actually possible to hit it with the current implemented methods because `chunks` is always provided. I could update this function to be less future-proof and not look for dask arrays and only look for chunk tuples.